### ISSUE TITLE
muse-combineAll Not called when not using grid submission

### ISSUE THREAD

**Issue by toomanycats** [CBICA/MUSE#2]
I've taken over a project from another staff which uses `muse` with ANTS and DRAMMS. The previous runs I inherited have this final output file:     
    
    *_thresholded_brain_muse.nii.gz

When I run the final muse fusion step, I'm not getting that output. I'm left with 152 files like this: 

    _thresholded_brain_ROI_*_SimRank+IC+FS.nii.gz

Previous jobs were run on our large memory machine and I've been running on a single core without our grid engine.
I suspect that the `muse-combineAll.sh` is not getting called.

**log output** Running on a normal host with `--NOQ`

```  $ tail  muse-labelFusion-11031.log
          --> Processing ROI 204
          --> Processing ROI 205
          --> Processing ROI 206
          --> Processing ROI 207
  -----> Moving the Rank Masks to /m/Dev/pijp-multiatlas/ADNI3_135_S_4722y02_i1180418/ ...
  -----> Creating a file list for combining the Rank Masks ...
  -----> Combining the Rank Masks ...
  -----> Removing temporary files from TMPDIR ...
  
  Execution time: 55m 25s```

Running on a GPU node with `-P 64 --NOQ`

```  $ tail muse-labelFusion-69804.log
          --> Processing ROI 204
          --> Processing ROI 205
          --> Processing ROI 206
          --> Processing ROI 207
  -----> Moving the Rank Masks to /m/Dev/pijp-multiatlas/ADNI3_135_S_4722y02_i1180418/ ...
  -----> Creating a file list for combining the Rank Masks ...
  -----> Combining the Rank Masks ...
  -----> Removing temporary files from TMPDIR ...
  
  Execution time: 12m 30s
```

This is my call:

            /opt/muse/muse-3.0.5/bin/muse -i {in_path} \
                                          --NOQ \
                                          --method 3 \
                                          -k 1 \
                                          --tmp {temp_path} \
                                          --log {log_path}""")

**Comment by toomanycats** [comment]
bump

**Comment by ashishsingh18** [comment]
@gurayerus @jammy270 Do you have any suggestions on this issue?

**Comment by toomanycats** [comment]
I've been able to get outputs by making two separate calls, one to `muse-combineAll` and another to `muse-calculateVolumes`.
I think this issue has something to do waiting for sub jobs to complete.

Shell scripts with this amount of complexity are difficult to debug.

**Comment by toomanycats** [comment]
I think I found the issue. I had to get the logging argument set.
I can see that we don't have a PE named 'threaded' and that causes a job to fail.

I found the SGE control scripts. I'll close this ticket if I can get it working. 
Thanks.

UPDATE: Now that we have a working PE the label fusion step seems to run but not completely. 
There's still no final `muse` output and no CSV files.

I'll try again with the `-P 4` flag. That's the only significant difference between our GPU runs and SGE runs.
